test.lua

```diff
@@ -0,0 +1,70 @@
+#!/usr/local/bin/tarantool
```

Comment:
I would prefer `#!/usr/bin/env tarantool`. For example, I have tarantool in `/usr/bin`, not `/usr/local/bin`, so current shebang would not work for me.

Reply:
Agree, will update once merged